src/ucp/rndv/rndv_mtype.inl

```diff
@@ -169,18 +169,24 @@ static UCS_F_ALWAYS_INLINE ucs_status_t ucp_proto_rndv_mtype_copy(
 
 static UCS_F_ALWAYS_INLINE void
 ucp_proto_rndv_mtype_query_desc(const ucp_proto_query_params_t *params,
+                                ucs_memory_type_t frag_mem_type,
                                 ucp_proto_query_attr_t *attr,
                                 const char *xfer_desc)
 {
     UCS_STRING_BUFFER_FIXED(strb, attr->desc, sizeof(attr->desc));
     ucp_context_h context      = params->worker->context;
     ucs_memory_type_t mem_type = params->select_param->mem_type;
-    ucp_ep_h mtype_ep          = ucp_proto_rndv_mtype_ep(params->worker,
-                                                         mem_type);
+    ucp_ep_h mtype_ep;
     ucp_lane_index_t lane;
     ucp_rsc_index_t rsc_index;
     const char *tl_name;
 
+    /* Make coverity happy */
+    ucs_assertv(frag_mem_type < UCS_MEMORY_TYPE_UNKNOWN, "frag_mem_type = %u",
+                frag_mem_type);
```

Comment:
1. maybe move this inside ucp_proto_rndv_mtype_ep ?
2. remove space before and after =